_layouts/sidebar.md

```diff
@@ -16,6 +16,7 @@
     - [Ayushi George](../journeys/ayushigeorge.md)
     - [Ebuka Chuqz](../journeys/ebukachuqz.md)
     - [Habeeb B. Oyesile](../journeys/badbatunde.md)
+    - [Hritesh Goldar](../journeys/hritesh94.md)
```

Comment:
Your link is automatically added by a github-action. Please remove it from here.

Reply:
ok did it and i have created another pr,hope everything is ok now